public/main.js

```diff
@@ -0,0 +1,102 @@
+/* global $ */
+
+function expressData(path) {
+  fetch(path)
+    .then(res => res.json())
+    .then(data => {
+      console.log(data)
+
+      const tableBody = document.createElement('tbody')
+
+      data.forEach(object => {
+        const tableRow = document.createElement('tr')
+        for (let key in object) {
+          const tableData = document.createElement('td')
+          tableData.textContent = object[key]
+          tableRow.appendChild(tableData)
+        }
+        tableBody.appendChild(tableRow)
+      })
+      console.log(tableBody)
+      table.appendChild(tableBody)
+    })
+}
+expressData('/events/new-york-mets')
```

Comment:
you don't want to make this call until the user chooses a team from the dropdown list